app/puzzle/[slug]/(components)/header/page-nav.tsx

```diff
@@ -25,7 +25,7 @@ const PuzzleHeaderPageNav: FC<PuzzleHeaderPageNavProps> = ({ prevPuzzle, nextPuz
   return (
     <ButtonGroup className="hidden sm:flex">
       <IconButton
-        className="z-[2]"
+        className={!nextPuzzle ? 'z-[2]' : 'z-[1]'}
```

Comment:
Why `z-[2]` not `z-[5]` or `z-[4]`?

Reply:
The only limit is that it must be less than 10:
![Uploading Screen Shot 2023-11-19 at 3.18.57 PM.png…]()

I chose to go with the smallest valid, positive index to minimize "collisions" down the line.

Reply:
There is a certain possibility that this can become conflicted on certain layered components, even though the default popover components can be of a higher `z-index`, maybe this is best not to be arbitrary or at least leave some typescript convention satisfying comments expressing the concerns

Reply:
# hPiw27AQYh0

![image](https://github.com/waterfall-mkt/curta.wtf/assets/66016924/0e89df4d-26bc-434f-9a5f-06112635f9bd)

Reply:
(specifically t=171)